specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -737,6 +737,7 @@ model TestRun {
   regionalStatistics?: Record<TestRunStatistics>;
 
   @doc("The load test configuration.")
+  @visibility("read")
```

Comment:
Service supports overriding subnetId and engineInstances. Will we change that as well?

Reply:
We are not targetting subnetId currently. engineInstances were never supported by the service.